x/gov/README.md

```diff
@@ -116,27 +120,26 @@ proposal is finalized (passed or rejected).
 #### Deposit refund and burn
 
 When a proposal is finalized, the coins from the deposit are either refunded or burned
-according to the final tally of the proposal:
+according to the final tally of the proposal. In Heimdall, burn is not enabled, hence all the deposits will be refunded.
```

Comment:
Perhaps instead of being refunded, tokens get sent to the fee collector so we uphold this behavior as a spam deterrent?

Reply:
Yeah this is actually a good point. Will discuss internally.